dataservice/__init__.py

```diff
@@ -72,9 +72,12 @@ def register_error_handlers(app):
     """
     Register error handlers
     """
-    pass
+    from dataservice.api import errors
+    from werkzeug.exceptions import HTTPException
+    app.register_error_handler(HTTPException, errors.http_error)
```

Comment:
This doesn't actually catch anything. Was supposed to be fixed recently in Flask, but couldn't get it to work. Might want to investigate more.

Reply:
Hmm interesting.. Well I like this idea of having a catch all error handler for anything we don't explicitly catch. 

I found this [post](https://stackoverflow.com/questions/29332056/global-error-handler-for-any-exception) that provides some options but maybe you already saw it?

Reply:
Yeah, I tried catching on `Exception` too, but something was getting in the way of the handler. Will need to investigate more.